third_party/move/move-binary-format/src/constant.rs

```diff
@@ -38,10 +38,9 @@ fn ty_to_sig(ty: &MoveTypeLayout) -> Option<SignatureToken> {
         MoveTypeLayout::Vector(v) => Some(SignatureToken::Vector(Box::new(ty_to_sig(v.as_ref())?))),
         MoveTypeLayout::Struct(_) => None,
         MoveTypeLayout::Bool => Some(SignatureToken::Bool),
-        MoveTypeLayout::Tagged(tag, layout) => match tag {
-            // Ignore aggregator / snapshot mappings.
-            LayoutTag::IdentifierMapping(_) => ty_to_sig(layout.as_ref()),
-        },
+
+        // It is not possible to have native layout for constant values.
```

Comment:
I don't follow the comment, what about constant values?

Reply:
This function is used to construct a type of a Move constant (i.e. `const ABC u64 = ...;`), which we store in the binary format (it is called `SignatureToken`). We do not expect constants to have types with delayed fields, unless we want custom serialization (maybe?), but making this an error is safer.

Reply:
Changed function name to make it clearer: `ty_to_sig`, what a great name it was